controllers/dpa_controller.go

```diff
@@ -146,16 +148,53 @@ func (r *DPAReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&corev1.Service{}).
 		Owns(&routev1.Route{}).
 		Owns(&corev1.ConfigMap{}).
-		Watches(&source.Kind{Type: &corev1.Secret{}}, &labelHandler{}).
+		Watches(&source.Kind{Type: &corev1.Secret{}}, &secretHandler{}).
 		WithEventFilter(veleroPredicate(r.Scheme)).
 		Complete(r)
 }
 
-type labelHandler struct {
+type secretHandler struct {
+	defauldSecrets map[string]*struct{}
+	mu             sync.RWMutex
+	client         client.Client
 }
 
-func (l *labelHandler) Create(evt event.CreateEvent, q workqueue.RateLimitingInterface) {
+func createSecretHandler() *secretHandler {
+	defaultSecrets := map[string]*struct{}{}
+	for _, fields := range credentials.PluginSpecificFields {
+		if fields.IsCloudProvider {
+			defaultSecrets[fields.SecretName] = nil
+		}
+	}
+	return &secretHandler{
+		defauldSecrets: defaultSecrets,
+		mu:             sync.RWMutex{},
+	}
+}
+
+func (l *secretHandler) Create(evt event.CreateEvent, q workqueue.RateLimitingInterface) {
 	// check for the label & add it to the queue
+	l.mu.RLock()
+	_, ok := l.defauldSecrets[evt.Object.GetName()]
+	l.mu.RUnlock()
+	if ok {
+		list := &oadpv1alpha1.DataProtectionApplicationList{}
+		err := l.client.List(context.Background(), list)
+		if err != nil {
+			//TODO: There should be a log here, and we probaly want to somehow re-queue the secret
+			return
+		}
+		for _, dpa := range list.Items {
+			q.Add(reconcile.Request{
```

Comment:
@shawn-hurley this is probably a dumb question. Are we not checking for labels anymore?

Reply:
The default secrets are hardcoded names, if we are watching all the secrets, lets just use the names was my thought